### PR TITLE
add scripts to download ABI reference atlas and convert to Nifti

### PR DESCRIPTION
**Pull request by setina42**

### CODE REVIEW

**Comment by setina42** [comment]
Will add Registration with Ants  and Resize tomorrow

**Review by TheChymera — CHANGES_REQUESTED** [review]
There's a lot of blank lines here. Please follow the guidelines of one blank line to delimit units of logic (2 blank lines only between functions which have blank lines instide --- of which there are none here), and no blank lines between a comment line and the code it comments, analogous to here: https://github.com/IBT-FMI/mouse-brain-atlases/blob/aaa5ef3982850d9b842749246e3bed83bf7d3c55/ambmc.sh

File: Nrrd_To_Nifti.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import nibabel
+import numpy as np
+import sys
+
+path = os.path.dirname(sys.argv[0])
+files = glob(path + '*.nrrd')
+
+for file in files:
+  print("Reading " + file)
+  readnrrd = nrrd.read(file)
```
Comment (TheChymera) [inline Nrrd_To_Nifti.py]: indent with tabs, not spaces.

**Review by TheChymera — CHANGES_REQUESTED** [review]

File: nrrd_to_nifti.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import nrrd
+import nibabel
+import numpy as np
+import sys
+
+path = os.path.dirname(sys.argv[0])
+files = glob(path + '*.nrrd')
+
+for file in files:
+  print("Reading " + file)
```
Comment (TheChymera) [inline nrrd_to_nifti.py]: Please indent with tabs, not spaces :)

**Review by TheChymera — CHANGES_REQUESTED** [review]

**Pull request merged**